File: 136-single-number/single-number.java
class Solution {
    public int singleNumber(int[] nums) {
        Arrays.sort(nums);
        int a=0;
        if(nums.length==1){
            return nums[0];
        }
        for(int i=0;i<nums.length;i++){
            int count=0;
            for(int j=0;j<nums.length;j++){
            if(nums[i]==nums[j]&&i!=j){
              count++;
            }
            }
            if(count==0){
                a=nums[i];
            }

           
        }
        return a;
    }
}

File: 136-single-number/single-number.py
class Solution:
    def singleNumber(self, nums: list[int]) -> int:
        d={}
        for num in nums:
            if  num not in d:
                d[num]=1
            else:
                d[num]+=1
        for num in nums:
            if d[num]==1:
                return num